src/libraries/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -180,7 +180,7 @@ public void OSVersion_ParseVersion(string input, int major, int minor, int build
             Assert.Equal(expected, actual);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotNativeAot))]
```

Comment:
This can be more specific - `if` around the `RuntimeIdentifier` check that I assume is failing.